idp/azure/azure.go

```diff
@@ -56,6 +56,11 @@ type Params struct {
 	// Hidden is set if the IDP should be hidden from interactive
 	// prompts.
 	Hidden bool `yaml:"hidden"`
+
+	//The Azure login URL
+	//For Azure AD it should be set to https://login.microsoftonline.com/${TENANT_ID} or https://sts.windows.net/${TENANT_ID}/
+	//Default to "https://login.live.com" (consumer account login)
+	LoginUrl string `yam:"loginurl"`
```

Comment:
Nitpick: in Go this would be spelled `LoginURL` and the documentation string would start with `LoginURL is the Azure login URL. For Azure AD...`.

Reply:
Indeed that is the proper way to spell variables and write godoc